doc/config.rst

```diff
@@ -71,3 +71,9 @@ configure them in your config file.
         ->setIndent("\t")
         ->setLineEnding("\r\n")
     ;
+
+Online configurator
```

Comment:
before adding any reference to external tool to configure the fixer, I would suggest we encourage the user to set up the config by using pre-existing rulesets listed in https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/ruleSets/index.rst , and if they want to go deeper and adjust it then to look at rules listed in https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/rules/index.rst ,

and then, if they really need sth very custom, use external configurator.

imho, first choice for 95% users should be predefined ruleset like PER or PSR.

Reply:
@keradus
>  if they really need sth very custom, use external configurator.

you can select pre-existing rulesets too:

![изображение](https://user-images.githubusercontent.com/3521094/195218049-228a1b2f-a630-43ae-94ef-fada97951dee.png)

It's more useful/faster than current wiki docs, you can generates **any** configuration without without having to remember classes using in `.php-cs-fixer.php` or searching fixers options in documentation

Reply:
mentioned online configurator is definitely very powerfull one, but it could also be overwhelming , when the first thing one see is list of ~300 rules

that's why I would first encourage to just "pick up the ruleset you need" and suggest few options for start, and then go deeper to configurator only if one really wants to customize the used rules.

Reply:
@keradus are not able to choose sets in the menu? or read hint (`?` button) understandable to a child?
Let users decide use this service or not.